package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "RepCrowdsourcer",
+	"name": "repcrowdsourcer",
```

Comment:
This change is surprising to see here.  This should match the string in `package.json`, but that isn't being changed in this PR.